notebooks/single_baseline_postprocessing_and_pspec.ipynb

```diff
@@ -7,7 +7,7 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.        if band_slice is None:
</pre><p>It is weird to me that you can have an inconsistent band_ind and band_slice. I'd suggest picking one or the other to specify.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera_notebook_templates/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>

Reply:
Just fixed this.